test/components/button_group_component_test.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
```

Comment:
What should we do if no buttons are declared?

Reply:
I guess not rendering is the best idea, will update